deps/prebuilt/include/TAEF/WexTestClass.h

```diff
@@ -115,13 +119,44 @@ extern "C" __declspec(noreturn) void __fastfail(unsigned int);
 #define TAEF_FAST_FAIL System::Environment::FailFast(nullptr)
 #endif // #if !defined(__cplusplus_cli)
 
+#if defined(__clang__)
+#include <array>
+#endif
+
 /// <summary>
 /// The WEX namespace contains classes used for marking up C++ Tests.
 /// </summary>
 namespace WEX
 {
     namespace Private
     {
+        #if defined(__clang__)
+        // tempalated struct using variadic template args to represent a sequence of characters
+        template <unsigned... CharacterIndices> 
+        struct index_sequence{};
+
+        // helper to convert N -> N, N-1, N-2, .. , 1 , 0 as template args
```

Comment:
> N -> N, N-1, N-2, .. , 1 , 0 [](start = 28, length = 29)

comment is actually backwards in terms of template arg order (correct in recursion order though). Probably just remove in favor of std::integer_sequence